Pr/cdr 2212 add switch component

KenjiCrosland opened the pull request.

mhewson commented on src/less/definitions/utilities/utilities.less:
```diff
@@ -270,132 +270,180 @@
+ .hidden-md {
+   @media (min-width: @screen-md-min) and (max-width: @screen-md-max) {
+     .responsive-invisibility();
+   }
```
don't these exist already in responsive-utilities.less?

wjramos replied:
don't remember what I was thinking here

wjramos commented:
it would be a good idea here to force `height: 0` to avoid unintended consequences

Commit:
feat(add switch component): initial commit

CDR-2212

Commit:
feat(remove failing test): remove failing test

CDR-2212

Commit:
feat(fix obsolete tests): fix obsolete tests

CDR-2212

Commit:
feat(cdr-switch): add switch token references. Remove unrelated test updates. Change component name

Add new switch tokens to the scss file. Remove popover and popup test updates (this will be in a
different PR). Change the name of the component from cdr-toggle-switch to just cdr-switch. Add size
style variants. Add slot capability for text label. Add more examples on example page.

CDR-2212

Commit:
feat(cdr-switch): add beta version of tokens to package.json and add tests

CDR-2212

Commit:
feat(cdr-switch): update chromedriver

CDR-2212

Commit:
feat(cdr-switch): update switch example id

CDR-2212

Commit:
feat(cdr-switch): convert px to rem. remove switch mixins

CDR-2212

Commit:
style(cdr-switch): add newlines

CDR-2212

Commit:
style(cdr-switch): add newline to scss file

Commit:
feat(cdr-switch): remove beta tag from cdr-tokens dependency

CDR-2212

benjag commented on src/components/switch/CdrSwitch.vue:
```diff
@@ -0,0 +1,65 @@
+    default: false
+  }
+})
+//Safari focus state
```
Delete?

Commit:
feat(cdr-switch): change switch to v-model

Update switch to use v-model. Update corresponding tests

Commit:
feat(cdr-switch): add switch component examples

CDR-2212

Commit:
feat(cdr-switch): add uid generator for uniqueIds

Commit:
Merge branch 'main' of github.com:rei/rei-cedar-next into pr/CDR-2212-add-switch-component

Commit:
feat(cdr-switch): update package-lock.json

CDR-2212

Commit:
feat(cdr-switch): update package-lock again

CDR-2212

Commit:
13.0.0-beta.8
